semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/operatorextension/package.scala

```diff
@@ -4,76 +4,82 @@ import gremlin.scala.GremlinScala
 import io.shiftleft.codepropertygraph.Cpg
 import io.shiftleft.codepropertygraph.generated.Operators
 import io.shiftleft.codepropertygraph.generated.nodes
+import io.shiftleft.codepropertygraph.generated.nodes.Call
+import io.shiftleft.overflowdb.OdbGraph
 
 package object operatorextension {
+  class Assignment(call: Call) extends Call(call.graph.asInstanceOf[OdbGraph], call.id)
```

Comment:
This should be `extends nodes.Call` instead of `extends Call`.

Reply:
I agree, but for whatever reason scalac needs it this way. I guess it's a weird edge case when using relative imports inside a package object, or whatever. I could debug this but don't think it's worth it, esp. given that we may rethink the naming of steps/nodes shortly.

Reply:
fwiw here's the scalac error, edge case doesn't seem to be the package object though. 
```
Error:(12, 26) stable identifier required, but Assignment.this.nodes found.
  class Assignment(call: nodes.Call) extends nodes.Call(call.graph.asInstanceOf[OdbGraph], call.id)
```